dnf-behave-tests/fixtures/specs/miscellaneous/minorbump-0:1.0-1.1.spec

```diff
@@ -0,0 +1,16 @@
+Name:           minorbump
```

Comment:
I think we need to sort out this `miscellaneous` situation :)

Reply:
Yeah :). I basically did not want to create new repos when I need only one package (although a special one)